modules/admin_manual/pages/enterprise/external_storage/windows-network-drive_configuration.adoc

```diff
@@ -348,14 +288,179 @@ processed.
+Type=simple
+StandardOutput=journal
+StandardError=journal
+SyslogIdentifier=%n
+KillMode=process
+RestartSec=1
+Restart=always
+....
+
+* Run once for each created file the following commands:
++
+[source,console]
+----
+sudo systemctl enable owncloud-wnd-listen-SERVER-SHARE.service
+sudo systemctl start  owncloud-wnd-listen-SERVER-SHARE.service
+----
+
+=== WND Process Queue Configuration
+
+Create or add a `crontab` file in `/etc/cron.d/oc-wnd-process-queue`.
+
+* Make a `crontab` entry to run a script iterating over all `SERVER SHARE` pairs with
+an appropriate `occ wnd:process-queue` command. The commands must be **strictly sequential**.
+This can be done by using `flock -n` and tuning the `-c` parameter of `occ wnd:process-queue`
+
+....
+* * * * *     sudo -u www-data /usr/bin/php /var/www/owncloud/occ wnd:process-queue <HOST> <SHARE>
+....
+
+
+=== Serializer
+
+Parallel runs of `wnd:process-queue` might lead to a user lockout.
+The reason for this is that several `wnd:process-queue` might use the
+same wrong password because it hasn’t been updated by the time they
+fetch it.
+
+As a result, it’s recommended to force the execution serialization of
+that command to prevent this issue. You might want to use
+http://www.thegeekstuff.com/2011/05/anacron-examples[Anacron], which
+seems to have an option for this scenario, or wrap the command with
+https://linuxaria.com/howto/linux-shell-introduction-to-flock[flock].
+
+If you need to serialize the execution of the `wnd:process-queue`, check
+the following example with
+https://linuxaria.com/howto/linux-shell-introduction-to-flock[flock]
+
+[source,console,subs="attributes+"]
+....
+flock -n /my/lock/file {occ-command-example-prefix} wnd:process-queue <host> <share>
+....
+
+In that case, flock will try get the lock of that file and won’t run the
+command if it isn’t possible. For our case, and considering that file
+isn’t being used by any other process, it will run only one
+`wnd:process-queue` at a time. If someone tries to run the same command
+a second time while the previous one is running, the second will fail
+and won’t be executed. Check https://linux.die.net/man/2/flock[flock’s documentation]
+for details and other options.
+
+
+== Troubleshooting
+
+If you encounter issues using Windows network drive,
+then try the following troubleshooting steps:
+
+First check the connection to the share by using
+https://www.samba.org/samba/docs/man/manpages-3/smbclient.1.html[smbclient]
+on the command line of the ownCloud server.
+
+Here is an example:
+[source,console,subs="attributes+"]
+....
+smbclient -U Username -L //Servername
+....
+
+Take the example of attempting to connect to the share named MyData
+using `occ wnd:listen`. Running the following command would work:
+
+[source,console,subs="attributes+"]
+....
+{occ-command-example-prefix} wnd:listen MyHost MyData svc_owncloud password
+....
+
+However, running this command would not:
```

Comment:
it might be better to remark that the command is case sensitive, and that it must match the information from the mount point configuration

Reply:
done